### DISCUSSION TITLE
What is the Plugin for?

### DISCUSSION THREAD

**Discussion by JasdanVM** [tr7zw/ItemSwapper#190] [Q&A]
What's the server side plugin exactly for?
Is it just for whitelisting the mod on that server?

**Comment by tr7zw** [comment]
No, clients can't get items out of shulkers on a normal server, the plugin enables being able to do that. Same with the restock function. As a side effect, the plugin also has the feature to disable the mod entirely on the server. Without the plugin or mod on the server side you can only swap items from inside your normal inventory(same as opening it, and hotkeying some item into the hotbar).

**Comment by JasdanVM** [comment]
Interestingly I have a mod that restocks my items when they run out, from the normal inventory at least.
